### DISCUSSION TITLE
Error since the switch to daylight saving time.

### DISCUSSION THREAD

**Discussion by MohamedEHJ** [vectordotdev/vrl#534] [Q&A] [answered]
Hi,
I got the "input is not enough for unique date and time" when i want to parse a custom timestamp:
```console
-> vector vrl --timezone Europe/Paris 'parse_timestamp!("20231029025935", format: "%Y%m%d%H%M%S")'
function call error for "parse_timestamp" at (0:58): Invalid timestamp "20231029025935": input is not enough for unique date and time
```

I think it's related to the timezone defined, but i dont understand the behavior

Thanks! 

cf discord issue: https://discord.com/channels/742820443487993987/1168498617842159626

**Comment by jszwedko** [comment, marked as answer]
Hi @MohamedEHJ !

The error here is correct. The input timestamp is not unique enough given that the timestamp you have there occurs twice as it falls within window where daylight savings time switches over for `Europe/Paris`. That is: there are two `20231029025935`s, an hour apart from each other.

**Reply by MohamedEHJ** [reply]
hi!

Thanks for the explanation, I understand! 
Is there a way to handle this case?

**Reply by jszwedko** [reply]
At which offset would you like to consider that timestamp?

**Reply by MohamedEHJ** [reply]
I will group the logs from the two-hour time slot starting at 2 a.m., so UTC +1
